Makefile

```diff
@@ -46,6 +46,8 @@ TARBALL:=node-problem-detector-$(VERSION).tar.gz
 # IMAGE is the image name of the node problem detector container image.
 IMAGE:=$(REGISTRY)/node-problem-detector:$(TAG)
 
+ADDITIONAL_BUILD_TAGS?=""
```

Comment:
I think this is not needed. Just using `BUILD_TAGS` is enough, right? We just need to append `journald` to whatever `BUILD_TAGS` the user specifies.

Reply:
Sounds good. Changed to `BUILD_TAGS`.